docs/spec.md

```diff
@@ -170,11 +174,64 @@ following constraints:
 * (C10) `size(scales) = size(zero_points)`.
 * (C11) If `is_empty(quantization_dimension)`, then `size(scales) = 1`.
 * (C12) `0 <= quantization_dimension`.
-
-At the moment, `QuantizationScale` is a floating-point constant, but there is
-strong interest in integer-based scales, represented with multipliers and
-shifts. We are planning to explore this in the near future
-([#1404](https://github.com/openxla/stablehlo/issues/1404)).
+* (C13) `0 <= multipliers`.
+* (C14) `0 < shifts`.
+
+All quantized tensor types, in a given StableHLO operation, either
+use scales or multipliers and shifts. The `scales`, `multiplers` and `shifts`,
+are related as:
+`round_nearest_even(scales... * 2^shifts...) = multipler`.
```

Comment:
Should "multiplier" on the right hand side be "multipliers..." to match the list of scales and shifts on the left?

Reply:
Thanks @dominicsymes!
fixed the typo from `multiplier` -> `multipliers`.
Regarding the `...`, per [link](https://github.com/openxla/stablehlo/blob/main/docs/spec.md#formulas)[B], it is used mainly in the context of `f(xs...)` to apply `f` on every element of `xs`. In this case, IMHO, it might not be needed as the `round_nearest_even(scales... * 2^shifts...)` with result in a list/tensor and it would suffice if the rhs `multipliers` is also of the same type. We have a related ticket https://github.com/openxla/stablehlo/issues/1589 to further tighten the notation to avoid any confusion around the use of `...`.

Reply:
Thanks for clarifying. I follow now why multipliers has no ... and that makes sense.